tests/test_widgets.py

```diff
@@ -185,6 +185,34 @@ class W(SuffixedMultiWidget):
         }, {}, 'price')
         self.assertEqual(result, ['1', 'lt'])
 
+    def test_value_omitted_from_data(self):
+        class A(SuffixedMultiWidget):
+            suffixes = ['_b']
+
+        a = A(widgets=[BooleanWidget])
+
+        result = a.value_omitted_from_data(['a', 'b', 'c'], './.', 'test')
+
+        self.assertIsNotNone(result)
+
+    def test_replace_name(self):
+        class A(SuffixedMultiWidget):
+            suffixes = ['test']
+
+        a = A(widgets=[BooleanWidget])
+
+        output = '<div name="test123_0"></div>'
+        index = 0
+        q = a.replace_name(output, index)
+        self.assertEqual(q, output.replace('0', 'test'))
+
+    def test_decompress_value_None(self):
```

Comment:
Is the capital `N` in the method name intended? I see lower `_none` a few times above.